modules/integration/registry/tests-deprecate/src/test/java/org/wso2/carbon/registry/smoke/test/GovernanceRegistrySmokeTestCase.java

```diff
@@ -387,7 +387,7 @@ private void smokeTestSubscriptions(String smokeTestCollection, String smokeTest
         InfoAdminServiceStub infoAdminServiceStub =
                 TestUtils.getInfoAdminServiceStub(loggedInSessionCookie);
         Assert.assertEquals(infoAdminServiceStub.subscribe(smokeTestCollection,
-                "https://localhost:9443/services/Hello", "CollectionUpdated",
+                "https://localhost:10343/services/Hello", "CollectionUpdated",
```

Comment:
Isn't there a better way to keep the port number? What if we have to change the port due to some reason?